randovania/game_connection/connector/dread_remote_connector.py

```diff
@@ -84,3 +140,56 @@ def new_inventory_received(self, json_string: str):
         self.last_inventory = inventory
         self.InventoryUpdated.emit(inventory)
 
+    @asyncSlot()
+    async def new_received_pickups_received(self, new_received_pickups: str):
+        new_recv_as_int = int(new_received_pickups)
+        self.logger.warning(f"Received Pickups: {new_received_pickups}")
```

Comment:
why warning?

Reply:
Probably some leftover from my tests way back.
Do we need this at all or should I just remove the line? Or more general: Do we want to keep any of the logging from this pr?

Reply:
Keep at first at debug level, so it shows in the log files.